astropy/io/fits/hdu/hdulist.py

```diff
@@ -930,27 +936,30 @@ def info(self, output=None):
         else:
             name = self._file.name
 
-        results = ['Filename: {}'.format(name),
-                   'No.    Name      Ver    Type      Cards   Dimensions   Format']
-
-        format = '{:3d}  {:10}  {:3} {:11}  {:5d}   {}   {}   {}'
+        file_line = 'Filename: {}\n'.format(name)
+        results = []
         default = ('', '', '', 0, (), '', '')
         for idx, hdu in enumerate(self):
-            summary = hdu._summary()
-            if len(summary) < len(default):
-                summary += default[len(summary):]
-            summary = (idx,) + summary
+            hdu_summary = hdu._summary()
+            summary = [idx, *hdu_summary, *default[len(hdu_summary):]]
             if output:
-                results.append(format.format(*summary))
+                results.append(tuple(map(str, summary)))
             else:
-                results.append(summary)
+                results.append(tuple(summary))
 
         if output:
-            output.write('\n'.join(results))
+            t = Table(rows=results, names=COLUMN_NAMES)
+            t_strings = t.pformat(max_lines=-1, max_width=-1, show_dtype=False,
+                                  align=COLUMN_ALIGN)
+            output.write(file_line)
+            # Avoid seperator line between names and content of the columns
+            output.write(t_strings[0])
+            output.write('\n')
```

Comment:
Is this really useful ? I don't see a blank line in the output.

Reply:
It's not a blank line. Its so the previous line contains a line-break. This is different from `print`, which contains `end='\n'` by default.

Reply:
Yes sorry, newline not blank line. But now I see, you're just removing line 1 :wink: